src/clipboard.cpp

```diff
@@ -210,7 +210,7 @@ void GraphicsWindow::PasteClipboard(Vector trans, double theta, double scale) {
             pt = pt.ScaledBy(fabs(scale));
             pt = pt.ScaleOutOfCsys(u, v, Vector::From(0, 0, 0));
             pt = pt.Plus(p);
-            pt = pt.RotatedAbout(n, theta);
+            pt = pt.RotatedAbout(n, -theta);
```

Comment:
All the (dead) mathematicians and physicists are spinning in their graves ;-)

Reply:
Oops, sorry for that, I never noticed it was the other way around. I didn't do too much CAD in my life. Will revert.